wrappers/python/tests/wnt/utils.py

```diff
@@ -68,7 +68,7 @@ def setup_log(module: str, level: str = "error"):
 
     logger = logging.getLogger(module)
     try:
-        logger.setLevel(ast.literal_eval("logging.{0}".format(level.upper())))
+        logger.setLevel(eval("logging.{}".format(level.upper())))
```

Comment:
Please use the following instead of eval:

```
        try:
            self._logger.setLevel(getattr(logging, self._level))
        except AttributeError:
            self._logger.setLevel(logging.DEBUG)
```

We faced the same at the gateway and backend client code  

https://github.com/wirepas/gateway/blob/7ec47dc66f4bcfda79a08da697528af2a5756c2e/python_transport/wirepas_gateway/utils/log_tools.py#L43

Reply:
Fixed